third_party/hedron_compile_commands/workspace.bzl

```diff
@@ -0,0 +1,15 @@
+"""Loads Hedron's Compile Commands Extractor for Bazel."""
+# https://github.com/hedronvision/bazel-compile-commands-extractor
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def repo():
+    http_archive(
+        name = "hedron_compile_commands",
+
+        # Replace the commit hash (0e990032f3c5a866e72615cf67e5ce22186dcb97) in both places (below) with the latest (https://github.com/hedronvision/bazel-compile-commands-extractor/commits/main), rather than using the stale one here.
+        # Even better, set up Renovate and let it do the work for you (see "Suggestion: Updates" in the README).
+        url = "https://github.com/hedronvision/bazel-compile-commands-extractor/archive/4f28899228fb3ad0126897876f147ca15026151e.tar.gz",
```

Comment:
can you put this commit / sha inside of workspace.bzl ?

Reply:
sure!